README.md

```diff
@@ -21,6 +21,17 @@ use caesarlib::*;
 
 ## Methods
 ```rust
+
+// With custom base sequence
+// Can look like "ABC...Zabc...z"
+
+custom_encipher(base_seq: &str, offset: u16, message: &str) -> String
```

Comment:
That sounds like a decent idea to prove that it works. Do you want to do it, or should I?